src/StepAggregator.php

```diff
@@ -192,4 +159,35 @@ public function getName()
     {
         return $this->name;
     }
+
+    /**
+     * Builds the pipeline
+     *
+     * @return callable
+     */
+    private function buildPipeline()
+    {
+        $nextCallable = function ($item) {
+            // the final callable is a no-op
+        };
+
+        $steps = clone $this->steps;
+
+        // Use illogically large and small priorities
+        // TODO: workaround SplPriorityQueue
+        $steps->insert(new Step\PcntlStep, PHP_INT_MAX);
```

Comment:
I would not use `PHP_INT_MAX`, some people may need to call a middleware before/after the standard middlewares are called. I think 100 should be the max. priority.